pkg/backporter/backporter.go

```diff
@@ -398,7 +403,7 @@ func getClonesTemplateData(bugID int, client bugzilla.Client, allTargetVersions
 	clonedReleases := sets.NewString()
 	for _, clone := range clones {
 		clone := clone
-		if len(clone.TargetRelease) > 0 {
+		if clone.TargetRelease != nil && clone.TargetRelease[0] != "---" {
 			majorRelease, err := getMajorRelease(clone.TargetRelease[0])
```

Comment:
This function is btw incorrectly named, sorry for not catching this earlier. Its `getMajorMinorRelease`